com.unity.cinemachine/Runtime/Behaviours/CinemachineConfiner2D.cs

```diff
@@ -296,7 +296,7 @@ static float CalculateHalfFrustumHeight(in CameraState state, in float cameraPos
                 frustumHeight = state.Lens.OrthographicSize;
             else
             {
-                // distance between the collider's plane and the camera
+                // distance between the collider plane and the camera
```

Comment:
I think this was ok. Or maybe `plane of the collider`.

Reply:
nobody cares, it's private in-line doc :)

Reply:
Anyway not public shouldn't have changed it mb